src/Components/SearchResults/SearchResults.jsx

```diff
@@ -34,21 +39,86 @@ const renderSecondary = (id, code, countryCode, ifopt) => {
  */
 function SearchResults(props) {
   const { currentSearchResults, processClickedResultHandler } = props;
+  const map = useSelector(state => state.MapReducer.olMap);
+  const [maxHeight, setMaxHeight] = useState(null);
+  const ListRef = useRef();
+  let olEventKey = null;
+
+  useEffect(() => {
+    olEventKey = map.on('change:size', () => updateMenuHeight());
+
+    return () => {
+      unByKey(olEventKey);
+    }
+  }, []);
+
+  useEffect(() => {
+    updateMenuHeight();
+  }, [currentSearchResults]);
+
+  const updateMenuHeight = () => {
+    let newMaxheight;
+
+    if (ListRef.current) {
+      const mapBottom = map.getTarget().getBoundingClientRect().bottom;
+      const elemRect = ListRef.current.getBoundingClientRect();
+      newMaxheight = mapBottom - elemRect.top - 35;
+    }
+
+    if (newMaxheight >= 0) {
+      setMaxHeight(newMaxheight);
+    }
+  };
+
   if (currentSearchResults.length === 0) {
     return null;
   }
   return (
-    <Paper square elevation={3}>
-      <List component="nav" aria-label="search results">
-        {currentSearchResults.map((searchResult, index) => {
-          if (index !== 0) {
+    <RootRef rootRef={ListRef}>
+      <Paper square elevation={3}>
```

Comment:
Why are you not using the ref property ? According to the doc RootRef seems close to be a deprecated component

Reply:
Clearly easier with ref prop directly of course :)